src/main/java/com/synopsys/integration/alert/channel/jira/actions/JiraGlobalTestAction.java

```diff
@@ -24,18 +24,58 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import com.google.gson.Gson;
+import com.synopsys.integration.alert.channel.jira.descriptor.JiraDescriptor;
 import com.synopsys.integration.alert.common.action.TestAction;
+import com.synopsys.integration.alert.common.exception.AlertException;
+import com.synopsys.integration.alert.common.persistence.accessor.FieldAccessor;
 import com.synopsys.integration.alert.common.rest.model.TestConfigModel;
 import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.jira.common.cloud.configuration.JiraServerConfig;
+import com.synopsys.integration.jira.common.cloud.configuration.JiraServerConfigBuilder;
+import com.synopsys.integration.jira.common.cloud.model.response.UserDetailsResponseModel;
+import com.synopsys.integration.jira.common.cloud.rest.JiraCloudHttpClient;
+import com.synopsys.integration.jira.common.cloud.rest.service.JiraCloudServiceFactory;
+import com.synopsys.integration.jira.common.cloud.rest.service.UserSearchService;
+import com.synopsys.integration.log.Slf4jIntLogger;
 
 @Component
 public class JiraGlobalTestAction extends TestAction {
     public static final Logger logger = LoggerFactory.getLogger(JiraGlobalTestAction.class);
+    private final Gson gson;
+
+    @Autowired
+    public JiraGlobalTestAction(final Gson gson) {
+        this.gson = gson;
+    }
 
     @Override
     public void testConfig(final TestConfigModel testConfig) throws IntegrationException {
-        logger.info("Test config invoked {}", testConfig);
+        final FieldAccessor fieldAccessor = testConfig.getFieldAccessor();
+        final JiraServerConfigBuilder jiraServerConfigBuilder = new JiraServerConfigBuilder();
+        final String jiraUrl = fieldAccessor.getString(JiraDescriptor.KEY_JIRA_URL).orElse(null);
+        final String accessToken = fieldAccessor.getString(JiraDescriptor.KEY_JIRA_ACCESS_TOKEN).orElse(null);
+        final String username = fieldAccessor.getString(JiraDescriptor.KEY_JIRA_USERNAME).orElse(null);
```

Comment:
Is this supposed to be a Username or an email address?

Reply:
I was not sure what to do about this. The username in Jira is the users email... from what I could tell. Our UI specifies this field as the username since it's what they use to log in. Should we change it all to email?